packages/core/upload/admin/src/pages/App/components/BulkMoveButton.tsx

```diff
@@ -1,16 +1,31 @@
-import React, { useState } from 'react';
+import * as React from 'react';
 
 import { Button, Modal } from '@strapi/design-system';
 import { Folder } from '@strapi/icons';
-import PropTypes from 'prop-types';
 import { useIntl } from 'react-intl';
 
-import { BulkMoveDialog } from '../../../components/BulkMoveDialog';
-import { AssetDefinition, FolderDefinition } from '../../../constants';
+import { BulkMoveDialog } from '../../../components/BulkMoveDialog/BulkMoveDialog';
 
-export const BulkMoveButton = ({ selected, onSuccess, currentFolder }) => {
+import type { File } from '../../../../../shared/contracts/files';
+import type { Folder as FolderDefinition } from '../../../../../shared/contracts/folders';
+
+interface FolderWithType extends FolderDefinition {
+  type: string;
+}
+
+interface FileWithType extends File {
+  type: string;
+}
```

Comment:
You export this type in BulkDeleteButton but then redfine it here in BulkMoveButton? Maybe it should be a in a common types file or at least imported from the other component instead of redefining.